File: week-1/problem-sets/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height();

int main(void)
{
    int height = get_height();
    for (int row = 0; row < height; row++)
    {
        for (int space = row + 1; space < height; space++)
        {
            printf(" ");
        }

        for (int block = height + row + 1; block > height; block--)
        {
            printf("#");
        }
        printf("\n");
    }
}

int get_height()
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);
    return h;
}

File: week-6/problem-sets/mario-more/mario.py
def main():
    height = int(get_height())
    for row in range(height):
        for space in range(row + 1, height):
            print(" ", end="")
        for block in range(height + row + 1, height, -1):
            print("#", end="")
        print("  ", end="")
        for block in range(height + row + 1, height, -1):
            print("#", end="")
        print()


def get_height():
    h = 0
    while not str(h).isdigit() or int(h) < 1 or int(h) > 8:
        h = input("Height: ")
    return h


if __name__ == "__main__":
    main()
